blog/2017-01-10-a-great-developer-experience.md

```diff
@@ -0,0 +1,44 @@
+authorFBID: 121800083
+---
+
+We strongly believe that great documentation is crucial to providing a great developer experience. The docs should be clear, concise, and useful to new users and veterans alike. With that in mind, we recently took some time to overhaul the Jest website.
+
+## Improved Docs
+
+One of the changes you'll notice upon visiting our docs is the updated sidebar. The documentation is now divided into three main areas: an introduction to Jest, advanced guides to Jest's features, and a comprehensive API reference.
+
+The **Introduction** section will guide you from installing Jest and writing your first case, to using Jest's matchers and testing async code. If you're new to Jest or need a quick refresher, these docs should get you up to speed in no time. If you've used Jest before and only need a quick reference on how it's installed, you need to go no further than the [Getting Started](/jest/docs/getting-started.html) guide.
+
+Once you feel comfortable using Jest, proceed to the advanced **Guides** section. The new [Snapshot Testing guide](/jest/docs/snapshot-testing.html) covers everything you need to know about creating and maintaining snapshot test cases.
+
+Finally, we've completely overhauled our API reference docs. You can now find detailed information on all of Jest's [Globals](/jest/docs/api.html), [matchers](/jest/docs/expect.html), and [every flag](/jest/docs/cli.html) supported by the `jest` CLI.
+
+## Who's Using Jest?
+
+We have created a [showcase of users](/jest/users.html) to highlight some of the companies that are using Jest. We're thankful to all of these companies for using Jest to test their websites, mobile apps, and APIs. If you're using Jest, check out the guidelines on GitHub and send us a pull request!
+
+<div class="miniShowcaseSection">
+ <div class="logos">
+   <img src="/jest/img/logos/twitter.png" title="Twitter"/>
+   <img src="/jest/img/logos/pinterest.png" title="Pinterest"/>
+   <img src="/jest/img/logos/paypal.png" title="PayPal"/>
+   <img src="/jest/img/logos/ibm.png" title="IBM"/>
+   <img src="/jest/img/logos/spotify.png" title="Spotify"/>
+ </div>
+</div>
+
+## Jest in the Browser
+
+As highlighted [last month](/jest/blog/2016/12/15/2016-in-jest.html), it is now possible to use Jest directly in the browser using [repl.it](https://repl.it/languages/jest). If you want to try out Jest before installing it, you can easily do so below or directly from the Jest homepage. Go ahead and give it a try!
+
+<iframe class="jest-repl" src="https://repl.it/languages/jest?lite=true"></iframe>
+
+## Get Involved
+
+This is just the start. Go ahead and take a look at the docs, and don't hesitate to send any feedback our way. If you find a mistake in the docs or you just want to let us know what needs to be documented better, please tweet at us at [@fbjest](https://twitter.com/fbjest) or [open an issue on GitHub](https://github.com/facebook/jest/issues) and tag it under "Documentation".
```

Comment:
can you send a follow up that asks users to send a PR through the edit button? :)